schema/RSINTEGRATEDWATERHEATER.schema.yaml

```diff
@@ -88,94 +88,64 @@ HeatSourceConfiguration:
       Data Type: "({RSRESISTANCEWATERHEATSOURCE},{RSCONDENSERWATERHEATSOURCE})"
       Constraints: "heat_source_type(RESISTANCE, CONDENSER)"
       Required: True
-    label:
-      Description: "Identifier for reference"
-      Data Type: "String"
-      Required: True
     heat_distribution:
-      Description: "Weighted distribution of heat by division, in order"
-      Data Type: "[Numeric]"
+      Description: "Weighted distribution of heat from the heat source along the height of the tank"
+      Data Type: "{WeightedDistribution}"
       Required: True
     turn_on_logic:
-      Description: ""
+      Description: "An array of logic conditions that define when the heat source turns on if it is currently off"
       Data Type: "[{HeatingLogic}]"
       Required: True
     shut_off_logic:
-      Description: ""
+      Description: "An array of logic conditions that define when the heat source shuts off if it is currently on"
       Data Type: "[{HeatingLogic}]"
-      Required: False
-    standby_logic:
+    standby_logic: # TODO: Is this needed?
```

Comment:
Lower priority. Needs to be researched to understand what it is intended to represent.

Reply:
If turn-on condition true, and standbyLogic exists, defers to standbyLogic to engage. If turn-on condition true, and no standbyLogic exists, engages, equivalent to top-node turn-on logic (perviously undefined). However, no IHPWH models have defined standbyLogic; this feature is only used by Sanco models. Other instances may have been coding remnants and are now replaced.

Reply:
Might be worth asking Ben if this is something that we need to keep moving forward, or if there is a simpler way to get the same outcome.